src/components/Questions/QuestionModal.vue

```diff
@@ -104,13 +104,13 @@ export default defineComponent({
      * triggered upon selecting an option
      */
     function questionOptionSelected(optionIndex: number) {
-      if (isQuestionTypeMCQ.value) {
+      if (isQuestionTypeSingleChoice.value) {
```

Comment:
Maybe too late to ask now, but why not use a single `questionType` variable (maybe array of constants) instead of a variable per question type?

Reply:
I guess it's just for ease of use
These are all computed variables per question type
We found it easier to work with rather than checking everytime what type the current question is

Reply:
Exactly what Deepansh said.